src/selectors/linesInScope.js

```diff
@@ -30,7 +30,10 @@ export default function getInScopeLines(state: OuterState) {
     source.toJS()
   );
 
-  const sourceNumLines = source.get("text").split("\n").length;
+  const sourceText = source.get("text");
+  const sourceNumLines = typeof sourceText === "string"
+    ? sourceText.split("\n").length
+    : sourceText.get("binary").length;
```

Comment:
* can we look at `source.isWasm` here instead of `typeof sourceText === "string"` 
* perhaps move this to a helper `utils/source`